src/main/resources/validationrules/core/organisational-role-mapping-common.drl

```diff
@@ -50,3 +51,24 @@ then
     update($ra);
     logMsg("Rule :: staff_organisational_role_mapping_service_delete");
 end;
+
+rule "sscs_system_user_hearings_roles_create"
+when
+    $rq: Request(
+             clientId == "sscs",
+             process == "sscs-system-users",
+             reference == "sscs-hearings-system-user",
+             replaceExisting == true)
+    $ra: RoleAssignment(
+             status == Status.CREATE_REQUESTED,
+             roleType == RoleType.ORGANISATION,
+             classification == Classification.PUBLIC,
+             roleName in ("hearing-manager","hearing-viewer"),
+             roleCategory == RoleCategory.SYSTEM,
+             grantType == GrantType.STANDARD)
```

Comment:
Hi Sankar  : How are we adding jurisdiction and granType here?

Reply:
Its in role_common.json file @nitinsahu1